integration-tests/keycloak-authorization/pom.xml

```diff
@@ -193,7 +193,7 @@
                         <configuration>
                             <images>
                                 <image>
-                                    <name>quay.io/keycloak/keycloak</name>
+                                    <name>quay.io/keycloak/keycloak:7.0.1</name>
```

Comment:
Couldn't we use a centralized property for that? It would have to be in the `build-parent` pom.xml.